.github/workflows/build.yaml

```diff
@@ -67,12 +64,36 @@ jobs:
             push: true
             tags: ${{ steps.metadata.outputs.tags }}
             labels: ${{ steps.metadata.outputs.labels }}
-        
-        - name: Push galasabld executables
+         
+        - name: Push Galasabld executables
           uses: actions/upload-artifact@v4
           with:
-            name: galasabld-${{matrix.config.name}}-${{matrix.config.arch}}
-            path: bin/galasabld-${{matrix.config.name}}-${{matrix.config.arch}}
+            name: galasabld-linux-amd64
+            path: bin/galasabld-linux-amd64
+
+        - name: Push Galasabld executables
+          uses: actions/upload-artifact@v4
+          with:
+            name: galasabld-windows-amd64
+            path: bin/galasabld-windows-amd64
+
+        - name: Push Galasabld executables
+          uses: actions/upload-artifact@v4
+          with:
+            name: galasabld-darwin-amd64
+            path: bin/galasabld-darwin-amd64
+
+        - name: Push Galasabld executables
+          uses: actions/upload-artifact@v4
+          with:
+            name: galasabld-darwin-arm64
+            path: bin/galasabld-darwin-arm64
+
+        - name: Push Galasabld executables
+          uses: actions/upload-artifact@v4
+          with:
+            name: galasabld-linux-s390x
+            path: bin/galasabld-linux-s390x

```

Comment:
Hi Dhruv, now that these are unique steps, these should be named uniquely so that it's clear from the view of the workflow exactly what is happening here. Can you update to "Publish galasabld-linux-amd64 executable" for example

Reply:
done